Cargo.toml

```diff
@@ -44,3 +44,7 @@ default = []
 all = ["serde", "cli"]
 serde = ["serde_crate", "serde_with", "lnpbp/serde", "bitcoin/use-serde", "rgb-std/serde", "amplify/serde", "chrono/serde"]
 cli = ["clap", "serde", "serde_yaml", "serde_json"]
+
+[patch.crates-io]
+# TODO: Remove after merge https://github.com/RGB-WG/rgb-std/pull/8
+rgb-std = { version = "0.8.0", git="https://github.com/crisdut/rgb-std/", branch="fix/opret1st" }
```

Comment:
Can you please pit this into a separate commit which can be removed before merging?